HW_6/task 6.2.1.js

```diff
@@ -0,0 +1,31 @@
+// Task 2
+// 1. У вас есть массив названий пицц вашего конкурента. Создайте функцию, которая будет принимать ваш набор названий пицц (массив) 
+//   и возвращать только те, которых нет у конкурента (тоже массив). Если все ваши пиццы есть у конкурента - вернуть null
+//   Пиццы конкурента:
+//   const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai']
+  
+const competitorPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai'];
+
+function checkPizzas(myPizzas) {
+    if (!Array.isArray(myPizzas)) {
+        throw new Error('Введите корректные данные');
+    }
+    
+    const result = [];
+    
+    for (const pizza of myPizzas) {       
+        if (!competitorPizzas.includes(pizza)){
+            result.push(pizza);
+        }
+    }
+    
+    if (result == '') {
+        return null;
+    } 
+    
+    return result;
```

Comment:
You've declared `result` as an array and after checking if it's empty string, hmmm
```suggestion
    return result.length ? result : null;
```